doc/modules/feature_extraction.rst

```diff
@@ -916,6 +916,33 @@ Some tips and tricks:
 
     (Note that this will not filter out punctuation.)
 
+
+    The following example will, for instance, transform some British spelling 
+    to American spelling::
+
+        >>> import re
+        >>> def to_british(tokens):
+        ...     for t in tokens:
+        ...         t = re.sub(r"(...)our$", r"\1or", t)
+        ...         t = re.sub(r"([bt])re$", r"\1er", t)
+        ...         t = re.sub(r"([iy])s(e$|ing|ation)", r"\1z\2", t)
+        ...         t = re.sub(r"ogue$", "og", t)
+        ...         yield t
+        ...
+        >>> class CustomVectorizer(CountVectorizer):
+        ...     def build_tokenizer(self):
+        ...         base = super(CustomVectorizer, self).build_tokenizer()
```

Comment:
one more thing, sorry!

could you rename this from `base` to `tokenize` or similar? (here as well as in the example)